src/test/java/htsjdk/samtools/util/HttpUtilsTest.java

```diff
@@ -0,0 +1,35 @@
+package htsjdk.samtools.util;
+import java.io.IOException;
+import java.net.URL;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import htsjdk.HtsjdkTest;
+
+public class HttpUtilsTest extends HtsjdkTest {
+    @DataProvider(name = "existing_urls")
+    public Object[][] testExistingURLsData() {
+        return new Object[][]{
+                {"http://broadinstitute.github.io/picard/testdata/index_test.bam"}
+        };
+    }
+    
+    @Test(dataProvider="existing_urls")
+    public void testGetHeaderField(final String url) throws IOException {
+    final String field = HttpUtils.getHeaderField(
```

Comment:
Indent is wrong here, it would be best to reformat the entire file since there are many lines with incorrect indent.